Update install and build documentation

cmcantalupo opened the pull request:
- Create a tab version of install guide
- Fixes #3709
- Rendering here:
   http://saruman.ra.intel.com/~cmcantal/install-doc/install.html

cmcantalupo committed:
Create a tab version of install guide

- OpenSUSE is actually openSUSE

Signed-off-by: Christopher M. Cantalupo <christopher.m.cantalupo@intel.com>

dannosliwcd commented on docs/source/install.rst:
```diff
@@ -1,265 +1,711 @@
+                     # SystemD service configuration and geopmd executable
+                     zypper install geopmd
+                     # C/C++ development files
+                     zypper install libgeopmd-devel
+                     # Man pages for GEOPM Access Service
+                     zypper install geopmd-doc
+                     # Man pages for C/C++ development
+                     zypper install libgeopmd-doc
+
+
+               .. group-tab:: Intel GPU support
```
Does "limited" just mean DRM-only? Maybe explain in the paragraph above the tab groups. Also what about NVML builds? e.g.,

> GEOPM works with Intel GPUs when GEOPM is built with LevelZero support and NVIDIA GPUs when GEOPM is built with NVML support. Additional limited support for GPUs exists when the GPUs are visible in a Linux Direct Rendering Manager (DRM) sysfs interface.

dannosliwcd commented on docs/source/install.rst:
```diff
@@ -1,265 +1,711 @@
-- ``libgeopmd2``: Provides the library that supports the PlatformIO interface.
-- ``python3-geopmdpy``: Houses the implementation of geopmd, CLI utilities, and bindings for PlatformIO.
-- ``geopm-service-devel``: Provides headers and man pages for C and C++ APIs supportable by ``libgeopmd2``.
+GEOPM Access Service - Official
```
Aren't all of our releases are equally official or unofficial? Maybe instead "stable" vs "development" packages?

cmcantalupo committed:
Add blurb about what GPU support means

- Official -> Stable
- Rolling -> Development

Signed-off-by: Christopher M. Cantalupo <christopher.m.cantalupo@intel.com>

cmcantalupo committed:
Add link to geopmaccess man and admin guide

Signed-off-by: Christopher M. Cantalupo <christopher.m.cantalupo@intel.com>

cmcantalupo committed:
Spell check and document package sources

Signed-off-by: Christopher M. Cantalupo <christopher.m.cantalupo@intel.com>

cmcantalupo committed:
Add links to service and runtime guides from install page

Signed-off-by: Christopher M. Cantalupo <christopher.m.cantalupo@intel.com>

cmcantalupo committed:
Update overview to simplify install section

- Update install to cover Spack use case
- Move Spack documentation into separate file

Signed-off-by: Christopher M. Cantalupo <christopher.m.cantalupo@intel.com>

cmcantalupo committed:
Add note about libgeopm dependency for geopmpy

Signed-off-by: Christopher M. Cantalupo <christopher.m.cantalupo@intel.com>

cmcantalupo committed:
Fix link and swich to 3.2.0 as latest in future commit

Signed-off-by: Christopher M. Cantalupo <christopher.m.cantalupo@intel.com>

cmcantalupo committed:
Move dev snapshot into separate doc

Signed-off-by: Christopher M. Cantalupo <christopher.m.cantalupo@intel.com>

cmcantalupo committed:
Add blurb about systemd and the admin guide

Signed-off-by: Christopher M. Cantalupo <christopher.m.cantalupo@intel.com>

cmcantalupo committed:
Remove build and install from service guide

Signed-off-by: Christopher M. Cantalupo <christopher.m.cantalupo@intel.com>

cmcantalupo committed:
Remove requires doc

Signed-off-by: Christopher M. Cantalupo <christopher.m.cantalupo@intel.com>

cmcantalupo committed:
Small updates for grammar and whitespace

Signed-off-by: Christopher M. Cantalupo <christopher.m.cantalupo@intel.com>

cmcantalupo committed:
Update build.rst to reflect new package names

- Add some examples for package.sh

Signed-off-by: Christopher M. Cantalupo <christopher.m.cantalupo@intel.com>

cmcantalupo committed:
Remove section about using Slurm plugin for save/restore

- This is obsoleted by the GEOPM Access Service daemon

Signed-off-by: Christopher M. Cantalupo <christopher.m.cantalupo@intel.com>

cmcantalupo committed:
Remove unneeded sudo

Signed-off-by: Christopher M. Cantalupo <christopher.m.cantalupo@intel.com>

bgeltz commented on docs/source/overview.rst:
Let's be consistent about "GEOPM Access Service" and "GEOPM Runtime Service" if we can.  The opening paragraph needs to be updated.

bgeltz replied:
"Listing Signal Information" needs to be updated as the behavior has changed:
```
[bgeltz@mcfly1 plan](issue-3674)$ geopmread -i CPU_FREQUENCY_STATUS
CPU_FREQUENCY_STATUS:
    description: The current operating frequency of the CPU.
    alias_for: MSR::PERF_STATUS:FREQ
    units: hertz
    aggregation: average
    domain: cpu
    iogroup: MSRIOGroup
[bgeltz@mcfly1 plan](issue-3674)$ geopmread -i MSR::PERF_STATUS:FREQ
MSR::PERF_STATUS:FREQ:
    description: The current operating frequency of the CPU.
    units: hertz
    aggregation: average
    domain: cpu
    iogroup: MSRIOGroup
```

Also, the example in "Breaking Down Signal/Control Names".
